### DISCUSSION TITLE
I’ve been approved for a few days and when can i receive my academic benefits?

### DISCUSSION THREAD

**Discussion by Kwen-Chen** [community/community#137910] [GitHub Education]
### Hi everyone,

 I’ve been approved for a few days and when can i receive my academic benefits? thanks!

**Comment by davevad93** [comment]
Hi @Kwen-Chen ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Kwen-Chen was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by joemar25** [comment]
Congratulations @Kwen-Chen on getting approved for your academic benefits! Typically, you can expect to receive your benefits within 3-7 business days after approval. However, during peak periods like the start of a new semester, it may take a bit longer
